docs/quickstart/installing.md

```diff
@@ -37,6 +37,36 @@ You can try lakeFS:
 1. With docker-compose [on Windows](more_quickstart_options.md#docker-on-windows).
 1. By [running the binary directly](more_quickstart_options.md#using-the-binary).
 
+## Modifying the local configuration file
+
+To modify the local configuration file, for example, in order to use your local lakeFS environment against S3 storage as opposed to the local storage, download the configuration file https://compose.lakefs.io, modify it and then run the container with the modified copy:
+
+```bash
+docker-compose -f ModifiedYMLFile.yml up
```

Comment:
1. We use the syntax `${LAKEFS_BLOCKSTORE_TYPE:-local}` in order to pass environment variables to the docker command without the need to download and edit it. So passing/expose the BLOCKSTORE/KEY/SECRET variables will do the job.
2. I think we need to add a note about the fact that lakeFS will try to identify repository that uses different storange namespace and prevert running - unless we remove the previous run and not just stop/start.

Reply:
@nopcoder ,
Would you say instead:
To modify the local deployment, for example, in order to use your local lakeFS environment against S3 storage as opposed to the local storage, you can either pass variables using the syntax ${LAKEFS_BLOCKSTORE_TYPE:-local}  or download the configuration file https://compose.lakefs.io, modify it and then run the container with the modified copy:

```bash
docker-compose -f ModifiedYMLFile.yml up
Member

And of course add your note that between runs you should remove the previous docker installation and not just stop/start it.

Reply:
Yes, just specify the names of the variable, not the complete bash syntax of the name+default